crates/fluvio-cli/src/connector/logs.rs

```diff
@@ -24,6 +25,23 @@ pub struct LogsManagedConnectorOpt {
 
 impl LogsManagedConnectorOpt {
     pub async fn process(self) -> Result<(), CliError> {
+        let config_file = FluvioConfigFile::load_default_or_new()?;
+        let fluvio_config = config_file.config();
+        let current_cluster = fluvio_config.current_cluster()?;
+        let endpoint = current_cluster.endpoint.clone();
+
+        if endpoint.ends_with("infinyon.cloud:9003") {
+            return Err(CliError::InvalidArg(
+                "fluvio connector logs does not work with fluvio-cloud".to_string(),
```

Comment:
I think that we should suggest the alternative: `fluvio cloud connector logs`?

Reply:
Should just check the host name to include 'cloud', not the port.